hexrd/ui/indexing/fit_grains_results_dialog.py

```diff
@@ -565,6 +573,55 @@ def reset_glyph_size(self, update_plot=True):
     def draw_idle(self):
         self.canvas.draw_idle()
 
+    def _save_workflow_files(self, selected_directory):
+        if selected_directory:
+            HexrdConfig().working_dir = selected_directory
+            HexrdConfig().save_indexing_config(
+                f'{str(selected_directory)}/workflow.yml')
+            HexrdConfig().save_materials(
+                f'{str(selected_directory)}/materials.h5')
+            HexrdConfig().save_instrument_config(
+                f'{str(selected_directory)}/instrument.yml')
```

Comment:
Removed the unnecessary string conversion.

For now the instrument is still being saved as a YAML file because there is an issue running find-orientations fails from the command line if the file is `.hexrd`. It looks like we'd need to update the hedm function [here](https://github.com/HEXRD/hexrd/blob/master/hexrd/config/instrument.py#L28) to support HDF5 files as well as YAML.

Reply:
Sounds good - thank you for finding this!